sources/core/Stride.Core/Native/NativeLibraryHelper.cs

```diff
@@ -56,6 +56,7 @@ public static void PreloadLibrary(string libraryName, Type owner)
                     Architecture.X86 => "x86",
                     Architecture.X64 => "x64",
                     Architecture.Arm => "ARM",
+                    Architecture.Arm64 => "arm64",
```

Comment:
Is it case-sensitive? I see `ARM64` in some other places.

Reply:
This ends up as the runtime identifier (RID), for example `win-arm64` - to my knowledge that's lower case. Why the upper entry isn't done that way I don't know.